app/api/routes_admin.py

```
@@ -0,0 +1,710 @@
+        want_debug = False
+
+    # Resolve runtime
+    start = time.perf_counter()
+    runtime = runtime_manager.get_active_runtime()
+    if runtime is None or (hasattr(runtime, "ready") and not getattr(runtime, "ready", True)):
+        raise_http_error(503, "runtime_unavailable", "model runtime is not available")
+
+    # perform prediction and map runtime errors to consistent envelopes
+    try:
+        tokens = await runtime.predict(text, timeout_ms=int(getattr(settings, "PREDICT_TIMEOUT_MS", 5000)))
+    except Exception as exc:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)